chains/rootstock/metadata.yaml

```
@@ -0,0 +1,31 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://rootstock.blockscout.com/api
+    family: blockscout
+    name: Blockscout
+    url: https://rootstock.blockscout.com
+blocks:
+  confirmations: 1
+  estimateBlockTime: 30
+  reorgPeriod: 4
+chainId: 30
+deployer:
+  # Deployment in discussion between Rootstock <> Abacus Works.
+  name: Abacus Works
+  url: https://www.hyperlane.xyz
+displayName: Rootstock
+domainId: 30
```

just one more thing to add
```suggestion
gasCurrencyCoinGeckoId: rootstock
```

---

Fixed!